cmd/foreachref.go

```diff
@@ -0,0 +1,51 @@
+package cmd
+
+import (
+	"flag"
+	"fmt"
+	"os"
+	"strings"
+
+	"github.com/driusan/dgit/git"
+)
+
+func ForEachRef(c *git.Client, args []string) error {
+	flags := flag.NewFlagSet("for-each-ref", flag.ExitOnError)
+	flags.SetOutput(flag.CommandLine.Output())
+	flags.Usage = func() {
+		flag.Usage()
+		fmt.Fprintf(flag.CommandLine.Output(), "\n\nOptions:\n")
+		flags.PrintDefaults()
+	}
+
+	// These flags can be moved out of these lists and below as proper flags as they are implemented
+	for _, sf := range []string{"format"} {
+		flags.Var(newNotimplStringValue(), sf, "Not implemented")
+	}
+
+	flags.Parse(args)
+	refs, err := git.ShowRef(c, git.ShowRefOptions{}, []string{})
```

Comment:
I think this part that does the work should be in the git package to be consistent with other commands.

Reply:
@sirnewton01 ping.. are you going to get a chance to refactor this, or do you want me to pick it up? I don't think it should be a lot of work.

Reply:
I can have a look at doing the refactor at some point. Sorry, just really busy at the moment.